sdk/test/trace/always_off_sampler_test.cc

```diff
@@ -0,0 +1,35 @@
+#include "opentelemetry/sdk/trace/always_off_sampler.h"
+
+#include <gtest/gtest.h>
+
+using namespace opentelemetry::sdk::trace;
+
+// Placeholder
+class MockSpanContext
+{}
+
+TEST(AlwaysOffSampler, MockSpanContext)
+{
+	Sampler sampler(new AlwaysOffSampler());
+
+	ASSERT_EQ("AlwaysOffSampler", sampler.GetDescription());
+
+	// Placeholder
+	MockSpanContext context(new MockSpanContext());
```

Comment:
You don't need to pass new MockSpanContext() when instantiating it on the stack. You can just leave it as 

MockSpanContext context();